### PR TITLE
Configure cloud provider integration tests

### PR DESCRIPTION
**Pull request by gravesm**
##### SUMMARY

This configures the AWS integration test target to run in CI and disables the Azure and GCP targets until we can configure credentials for those. It also swaps out the steampunk.aws collection for the amazon.aws collection and adds the necessary test dependency files.

Depends-On: https://github.com/ansible/ansible-zuul-jobs/pull/1749

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by gravesm** [comment]
recheck

**Comment by gravesm** [comment]
recheck

**Comment by gravesm** [comment]
recheck

**Comment by gravesm** [comment]
recheck

**Comment by gravesm** [comment]
recheck

**Comment by gravesm** [comment]
recheck

**Comment by gravesm** [comment]
recheck

**Comment by jillr** [comment]
recheck

**Comment by jillr** [comment]
recheck

**Comment by jillr** [comment]
recheck

**Comment by jillr** [comment]
recheck

**Comment by jillr** [comment]
recheck

**Comment by jillr** [comment]
recheck

**Comment by jillr** [comment]
recheck

**Comment by jillr** [comment]
recheck

**Comment by jillr** [comment]
recheck

**Comment by jillr** [comment]
recheck

**Comment by jillr** [comment]
recheck

**Comment by jillr** [comment]
recheck

**Comment by gravesm** [comment]
recheck

**Comment by gravesm** [comment]
recheck

**Comment by gravesm** [comment]
recheck

**Comment by gravesm** [comment]
recheck

**Comment by gravesm** [comment]
recheck

**Comment by jillr** [comment]
recheck

**Comment by jillr** [comment]
recheck

**Comment by jillr** [comment]
recheck

**Comment by jillr** [comment]
recheck

**Comment by jillr** [comment]
recheck

**Comment by jillr** [comment]
recheck

File: tests/integration/targets/aws/tasks/main.yml [hunk 1]
```diff
@@ -2,133 +2,132 @@
...
 # SPDX-License-Identifier: GPL-3.0-or-later
+- environment:
+    AWS_ACCESS_KEY_ID: "{{ aws_access_key | default(omit) }}"
+    AWS_SECRET_ACCESS_KEY: "{{ aws_secret_key | default(omit) }}"
+    AWS_SESSION_TOKEN: "{{ security_token | default(omit) }}"
+    AWS_REGION: "{{ aws_region | default(omit) }}"
+  block:
+    - set_fact:
+        test_basedir: "{{ test_basedir | default(output_dir) }}"
+        resource_id: "{{ '00' | community.general.random_mac | hash('md5') }}"
```
[the remark this answers is not in the document]
Comment (gravesm) [reply tests/integration/targets/aws/tasks/main.yml]: Good call, I just did this.

**Review by jillr — APPROVED** [review]

**Pull request merged**